Samples_for_Extending_Pentaho/javascript-apis/platform/visual-samples-bar-d3/.npmignore

```diff
@@ -1,3 +1,2 @@
 *.tgz
 sandbox.html
-sample-data/*
```

Comment:
Yep, João Gameiro noticed that today. But should be replaced with `sandbox-data.json`, not just deleted.